src/test/java/tests/AbstractTest.java

```diff
@@ -0,0 +1,97 @@
+package tests;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.TestExecutionExceptionHandler;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.ie.InternetExplorerDriver;
+import io.github.bonigarcia.wdm.WebDriverManager;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+
+public abstract class AbstractTest implements TestExecutionExceptionHandler {
+
+    protected WebDriver driver;
+    protected Properties envProperties;
+
+    @BeforeEach
+    public void setUp() {
+        configureDriver();
+        envProperties = loadProperties(".env-test");
+    }
+
+    private Properties loadProperties(String filePath) {
+        Properties properties = new Properties();
+        try (FileInputStream fileInputStream = new FileInputStream(filePath)) {
+            properties.load(fileInputStream);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
is it expected that execution continues in case of exception?

Reply:
No, it's not. Added throwing RuntimeException there